### ISSUE TITLE
Frozen shoulder

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7505]
**Preferred term label:**
Frozen shoulder

**Synonyms**
Adhesive capsulitis; Primary idiopathic stiff shoulder; FS.

**Definition (free text, please give PubMed ID)**
Inflammatory process occurring in the shoulder that leads to a painful and decrease or mechanical block in active and passive range of shoulder motion (PMID: 34772852; 33912325).

**Parent term (use hpo.jax.org/app)**
Stiff shoulders HP:0009742

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
I think that this (Adhesive capsulitis) is actually a disease and should go in MONDO @nicolevasilevsky 
See -- https://pubmed.ncbi.nlm.nih.gov/34772852/
https://github.com/monarch-initiative/mondo/issues/5366
